brion/blueConfig.cpp

```diff
@@ -213,16 +213,14 @@ class BlueConfig
     Strings names[CONFIGSECTION_ALL];
     ValueTable table[CONFIGSECTION_ALL];
 };
-}
+} // namespace detail
 
 BlueConfig::BlueConfig(const std::string& source)
     : _impl(new detail::BlueConfig(source))
 {
 }
 
-BlueConfig::~BlueConfig()
-{
-}
+BlueConfig::~BlueConfig() = default;
```

Comment:
Does it make sense to put this in the header? (I don't know what style you guys use)

Reply:
Maybe, I am not sure if there is a reason for having it here or in the header, so I will keep it for this PR.